data/data/alibabacloud/cluster/vpc/slb.tf

```diff
@@ -18,9 +18,6 @@ resource "alicloud_slb_listener" "listener_external_6443" {
   backend_port              = 6443
   frontend_port             = 6443
   protocol                  = "tcp"
-  bandwidth                 = 10
-  sticky_session            = "on"
```

Comment:
@bd233 if we don't want session enabled and it can cause issues when enabled, I believe we should explicitly disable it, even if the default is off.

Reply:
Thanks. Have  updated.